python/cuxfilter/charts/core/core_chart.py

```diff
@@ -203,6 +203,11 @@ def _yaxis_stride_type_transform(self, stride_type):
         """
         return dt.transform_stride_type(stride_type, self.y_dtype)
 
+    def _get_min_max(self, df, col_name):
+        if isinstance(df, dask_cudf.core.DataFrame):
+            return (df[col_name].min().compute(), df[col_name].max().compute())
+        return (df[col_name].min(), df[col_name].max())
```

Comment:
Suggest avoid dupe code inside the method:
```
min, max = df[col_name].min(), df[col_name].max()

if isinstance(df, dask_cudf.core.DataFrame):
    return (min.compute(), max.compute())

return (min, max)
```

But also suggest just making this a free util function, not a method (then `core_widget.py`) won't need an identical method)

Reply:
fixed, added it as a free util function under assets.cudf_utils